lib/resources/azure/azure_virtual_machine_data_disk.rb

```diff
@@ -17,8 +17,6 @@ class AzureVirtualMachineDataDisk < AzureResourceBase
     filter = FilterTable.create
     filter.add_accessor(:where)
           .add_accessor(:entries)
-          .add_accessor(:has_data_disks?)
-          .add_accessor(:count)
```

Comment:
Can you explain why we are removing these?

Reply:
Looks like they are also defined as methods - see lines 64 and 69.  Adding them as accessors would indeed trigger a method redefined warning.